tools/cli/internal/apiversion/version.go

```diff
@@ -150,9 +167,17 @@ func (v *APIVersion) IsPreview() bool {
 	return IsPreviewSabilityLevel(v.version)
 }
 
+func (v *APIVersion) IsPrivatePreview() bool {
+	return strings.Contains(v.version, PrivatePreviewStabilityLevel)
```

Comment:
Why don't you call `IsPreviewSabilityLevel(v.version)` here?

Reply:
IsPreviewSabilityLevel returns true to either Public or Private previews so we need to check for private preview here, but open to other suggestions